test/linalg/bidiag.jl

```diff
@@ -309,18 +309,18 @@ end
     @test promote(C,A) isa Tuple{Tridiagonal, Tridiagonal}
 end
 
-import Base.LinAlg: fillslots!, UnitLowerTriangular
-@testset "fill! and fillslots!" begin
+import Base.LinAlg: fillstored!, UnitLowerTriangular
+@testset "fill! and fillstored!" begin
     let #fill!
```

Comment:
This enclosing `let` seems vestigial; perhaps remove and de-indent while touching this code?